services/web-app/components/dashboard/dashboard.module.scss

```diff
@@ -91,6 +91,19 @@
   @include type.type-style('fluid-heading-04', true);
 }
 
+.framework-icon {
+  position: relative;
+  top: -(spacing.$spacing-02);
```

Comment:
```suggestion
   top: convert.rem(-3px);
```

Reply:
as well as this at the top `@use '@carbon/layout/scss/convert';`

This is super minor but the svg looked a bit more aligned like this - let me know what you think! 

Previous: 
<img width="419" alt="Screen Shot 2022-08-09 at 12 30 44 PM" src="https://user-images.githubusercontent.com/32720851/183721501-8b2fa9b9-17b1-4d7a-9803-8a8867395772.png">

1px. less:
<img width="363" alt="Screen Shot 2022-08-09 at 12 30 25 PM" src="https://user-images.githubusercontent.com/32720851/183721506-afc619dc-0cdc-42df-bd87-859d1ff58bf2.png">